cookbooks/OpenAI-ChatCompletion-AIConfigWrapper/openai_wrapper.ipynb

```diff
@@ -677,7 +941,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.11.6"
+   "version": "3.10.13"
```

Comment:
We should have a cell showcasing the wrapper with streaming as well. The rest of the notebook lgtm

Reply:
I was a little unclear why that's an important thing to demo here. Isn't it just about creating an aiconfig from existing code?

Reply:
Yes that's correct. Its important because it showcases to the user that this wrapper works with complex code that is required for streaming.